asv/benchmark.py

```diff
@@ -548,8 +548,9 @@ def do_setup(self):
         try:
             for setup in self._setups:
                 setup(*self._current_params)
-        except NotImplementedError:
+        except NotImplementedError as e:
             # allow skipping test
+            print("Skipped: {!r} ".format(e.message if e.message else "no message provided."))
```

Comment:
`"asv: skipped: {!r}"` so it can be distinguished from messages printed by user code